### PR TITLE
feat(icons): add new icons set

### PR DESCRIPTION
**Pull request by Alexandre-Herve**
- ActivityLogIcon
- DashboardIcon
- SharingGroupIcon
- UserCircleIcon

### CODE REVIEW

**Comment by pducolin** [comment]
Hi Alexandre! Did you follow [this guide](https://github.com/Dashlane/ui-components/blob/master/documentation/ICON_STEPBYSTEP.md)? I can't see the changes in the icons-def.json

**Comment by Alexandre-Herve** [comment]
@pducolin no I did not find it :facepalm: I was looking for something inside the icons folder.
Will do.

**Comment by Alexandre-Herve** [comment]
It's a bit complicated though, I wonder if we could have something a bit more automatic

**Comment by Alexandre-Herve** [comment]
Also if I look at the neighbours, the " are not escaped, they are replaced by single quotes

**Comment by Alexandre-Herve** [comment]
I've added the icons to the design-tokens @pducolin, all good now!

File: src/atoms/icons/ActivityLogIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { SvgWrapper } from '@ions/icon/wrapper/SvgWrapper';
+import { IconProps } from '@ions/icon/types';
+
+/*
+ * /!\ This is a generated file, do not update it or create a similar one by hand.
+ * Please refer to the doc in `documentation/ICON_STEPBYSTEP.md` to learn how to deal with icons.
+ */
```
Comment (oclero) [inline src/atoms/icons/ActivityLogIcon.tsx:8]: A big YES

**Pull request merged**